python/vyos/ifconfig/wireguard.py

```diff
@@ -24,7 +24,7 @@
 
 from vyos.ifconfig import Interface
 from vyos.ifconfig import Operational
-from vyos.template import is_ipv6
+from vyos.template import is_ipv6, is_ipv4
```

Comment:
Please move this to a new line - it makes search/replace with patterns more easy.